### ISSUE TITLE
Add OpenMM version number to API docs?

### ISSUE THREAD

**Issue by jchodera** [openmm/openmm#3402]
I've just realized that the [latest API docs](http://docs.openmm.org/latest/api-python/) don't mention which version of OpenMM they are for, and methods and classes don't indicate which version they were introduced in, so it's difficult to identify which version of the API docs you are working with. 

We might be able to improve this minimally by including the OpenMM version in the [API docs main page](http://docs.openmm.org/latest/api-python/).

**Comment by jchodera** [comment]
@peastman: Have you been able to confirm that the docs have actually updated for the OpenMM 7.7 release version?
https://openmm.org/documentation

**Comment by peastman** [comment]
Yes, I updated them a while ago.  For example, if you look at http://docs.openmm.org/latest/userguide/application/02_running_sims.html#force-fields you'll see it includes the new force fields that were added in 7.7.
